integration/substrate/ink/caller/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "caller"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
```

Comment:
Maybe either remove the line or fill it in. It's a test so not terribly important